gravitee-am-service/src/main/java/io/gravitee/am/service/model/UpdateIdentityProvider.java

```diff
@@ -37,45 +42,7 @@ public class UpdateIdentityProvider {
 
     private List<String> domainWhitelist;
 
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
-
-    public String getConfiguration() {
-        return configuration;
-    }
-
-    public void setConfiguration(String configuration) {
-        this.configuration = configuration;
-    }
-
-    public Map<String, String> getMappers() {
-        return mappers;
-    }
-
-    public void setMappers(Map<String, String> mappers) {
-        this.mappers = mappers;
-    }
-
-    public Map<String, String[]> getRoleMapper() {
-        return roleMapper;
-    }
-
-    public void setRoleMapper(Map<String, String[]> roleMapper) {
-        this.roleMapper = roleMapper;
-    }
-
-    public List<String> getDomainWhitelist() {
-        return domainWhitelist;
-    }
-
-    public void setDomainWhitelist(List<String> domainWhitelist) {
-        this.domainWhitelist = domainWhitelist;
-    }
+    private String passwordPolicy;
```

Comment:
**discussion**: According to the current specification/figma design, the linked between IDP and Policy is done in the policy screen where we can select multiple IDP. I'm not sure that we have to expose this capability to the UpdateIdentityProvider for now.

Reply:
Right.